tests/pds/dataservice_test.go

```diff
@@ -1049,7 +1049,7 @@ var _ = Describe("{DeletePDSEnabledNamespace}", func() {
 			for _, ds := range params.DataServiceToTest {
 				Step("Deploy and validate data service", func() {
 					isDeploymentsDeleted = false
-					deployment, _, _, err = DeployandValidateDataServices(ds, params.InfraToTest.Namespace, tenantID, projectID)
+					deployment, _, _, err = DeployandValidateDataServices(ds, nname, tenantID, projectID)
 					log.FailOnError(err, "Error while deploying data services")
```

Comment:
Make this error message more elaborate. Error while failing what kind of data service?
Helps in debugging.

Reply:
Done